packages/react-wildcat/cli/wildcat.js

```diff
@@ -19,6 +19,7 @@ program
     .parse(process.argv);
 
 const server = cp.spawn("node", [
+    "--debug",
```

Comment:
Instead of hard-coding this, I wonder if we can accept CLI params from the user instead. For example, if a user passes:

```
wildcat -- --debug --trace_gc
```

The CLI would parse out everything after the double dash (`--`), and pass it directly to Node.